Creds_AV_Update/README.md

```diff
@@ -0,0 +1,19 @@
+
+# AV Update Template
+
+**Objective:** Credential Harvesting
+
+## Pretext:
+Spoofing as an IT administrator, you're pushing out new antivirus software and need users to help run an automated scan to see if the new software was pushed to their computer and is up-to-date.
+
+## Things you'll need to change:
+* File: ./login: 
+  ** name="logo" -> update to the company logo
```

Comment:
This this formatting is invalid markdown, it's not rendering correctly. I think the sub-bullets just need another indentation like:
```markdown
* File: ./login:
  * name="logo" -> update the company logo
```
Also putting the code in back ticks will help differentiate it from the rest of the text.